setup_wifi.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python3
+
+# Script for changing MultiAP WiFi settings in Teltonika RUTX11.
+# Requires RUTX11 device with configuration set by first_setup.sh script - by default done by Husarion during production.  
+
+import io
+import json
+from pssh.clients import SSHClient
+import pssh.exceptions 
+import subprocess
+import time
+import os
+import sys
```

Comment:
```suggestion
import io
import json
import os
from pssh.clients import SSHClient
import pssh.exceptions 
import subprocess
import sys
import time
```
Lest keep alphabetical order